packages/storage/src/utils/pathUtils.ts

```diff
@@ -0,0 +1,16 @@
+/**
+ * Cleans up the path input to ensure it confirms
+ * to what textile buckets is expecting.
+ *
+ * @param path
+ */
+export const sanitizePath = (path: string): string => {
```

Comment:
Maybe we can use regex util that converts win -> posix paths https://github.com/sindresorhus/slash/blob/master/index.js?

Reply:
good idea, I'll update it.

Reply:
on second thought, I looked at the library and it appears to be using the same logic as used below, so maybe it is not worth adding that library. Especially since I'll have to be defining types for it to work well with typescript.